spec/git-spec.coffee

```diff
@@ -453,6 +453,24 @@ describe "git", ->
       it 'returns the status of the given path', ->
         expect(repo.getStatus('a.txt')).toBe 1 << 9
 
+  describe '.getStatusForPaths([paths])', ->
+    beforeEach ->
+      repoDirectory = temp.mkdirSync('node-git-repo-')
+      wrench.copyDirSyncRecursive(path.join(__dirname, 'fixtures/master.git'), path.join(repoDirectory, '.git'))
+      repo = git.open(repoDirectory)
+
+      newDir = path.join(repo.getWorkingDirectory(), 'secret-stuff')
+      fs.mkdirSync(newDir)
+
+      newFilePath = path.join(newDir, 'b.txt')
+      fs.writeFileSync(newFilePath, '', 'utf8')
+
+    describe 'when a path is specified', ->
+      it 'returns the status of only that path', ->
+        statuses = repo.getStatusForPaths(['secret-stuff'])
```

Comment:
What do you think about adding a few other assertions for this new API?

`repo.getStatusForPaths()`
`repo.getStatusForPaths([])`
`repo.getStatusForPath([''])`

Just to ensure none of these cause hard crashes on the C++ side of things.